docs/kyma/docs/031-gs-local-installation.md

```diff
@@ -15,35 +15,38 @@ To run Kyma locally, clone this Git repository to your local machine and downloa
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.10.0
 - [Helm](https://github.com/kubernetes/helm) 2.8.2
 - [jq](https://stedolan.github.io/jq/)
-- [Hyperkit driver](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#hyperkit-driver) - Mac only
-- [Virtualbox](https://www.virtualbox.org/) - Linux or Windows
+
+Vitrualization:
+
+- [Hyperkit driver](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#hyperkit-driver) - Mac
+- [Virtualbox](https://www.virtualbox.org/) - Linux, Windows, Mac
 - Hyper-V - Windows
```

Comment:
maybe add link to https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v ?

Reply:
sure